Content.Shared/_RMC14/Xenonids/Parasite/SharedXenoParasiteSystem.cs

```diff
@@ -297,6 +318,8 @@ private void OnParasiteLeapDoAfter(Entity<XenoParasiteComponent> ent, ref XenoLe
 
     private void OnParasiteLeapStopped(Entity<XenoParasiteComponent> ent, ref XenoLeapStoppedEvent args)
     {
+        RemComp<PreventCollideComponent>(ent);
```

Comment:
This might get run during state application, and if it does it will make the client spam refresh PVS if its removed instantly instead of deferred
```suggestion
        RemCompDeferred<PreventCollideComponent>(ent);
```